notebooks/WET/WET-003/WET-003-ComCam.ipynb

```diff
@@ -7,217 +7,757 @@
    "source": [
```

Comment:
<p>typo: "replace <strong>one</strong> of the corner sensors"</p><p> </p><p>Also,  clarify last sentence, I'm not sure I understand it.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>